core/store/src/db.rs

```diff
@@ -201,9 +203,6 @@ pub enum DBCol {
     ColStateChangesForSplitStates = 49,
 }
 
-// Do not move this line from enum DBCol
-pub const NUM_COLS: usize = 50;
```

Comment:
seems like a footgun to move this to top? I think it's intended that that's next to 49?